script/generate_sql.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/python
```

Comment:
please format this file with [black](https://github.com/python/black#installation-and-usage), make it executable, remove the `.py` extension, and use `/usr/bin/env python3` for the shebang:
```suggestion
#!/usr/bin/env python3
```